include/wx/qt/glcanvas.h

```diff
@@ -87,4 +87,7 @@ class WXDLLIMPEXP_GL wxGLCanvas : public wxGLCanvasBase
     wxDECLARE_CLASS(wxGLCanvas);
 };
 
```

Comment:
@bdbcat  There is a request to move any includes not required by wxWidgets users out of the wx header files and down into the implementation file.  Are these 2 includes needed by end users?

Reply:
Dan...
Need more context.  What two include files, what source file?

Reply:
There are a bunch of files in the PR coming in from the submodule libraries (expat, pnt, tiff).  Look at the file compare at github.com (can't see it in git diff probably).